spec/services/atmosphere/optimizer_spec.rb

```diff
@@ -115,8 +115,8 @@
                                   cpu: biggest_os_flavor.cpu,
                                   hdd: biggest_os_flavor.hdd,
                                   tenant: amazon)
-          biggest_os_flavor.set_hourly_cost_for(Atmosphere::OSFamily.first, 100)
-          optimal_flavor.set_hourly_cost_for(Atmosphere::OSFamily.first, 100)
+          biggest_os_flavor.set_hourly_cost_for(Atmosphere::OSFamily.first, 1)
+          optimal_flavor.set_hourly_cost_for(Atmosphere::OSFamily.first, 1)
```

Comment:
The old selection logic was bogus. It preferred a flavor with price 100 on amazon over a flavor with price 80 on openstack even when both flavors satisfied AT requirements and the AT had VMTs on both tenants.

This was never caught because (1) the spec was also bogus (hence the change); (2) the situation never came up on PL-Grid.

Btw., I did not write the old selection logic. ;)